From: SeongJae Park <sj@kernel.org>
Date: Mon, 6 Oct 2025 10:14:42 -0700
Subject: [PATCH] Docs/admin-guide/mm/damon/stat: fix
 damon_stat_monitoring_accuracy_overhead label

The label is missing the ending colon.  Fix it.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202510060124.YaUaLfkH-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/admin-guide/mm/damon/stat.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/admin-guide/mm/damon/stat.rst b/Documentation/admin-guide/mm/damon/stat.rst
index cfd7bf369e46..72123194f8e7 100644
--- a/Documentation/admin-guide/mm/damon/stat.rst
+++ b/Documentation/admin-guide/mm/damon/stat.rst
@@ -10,7 +10,7 @@ on the system's entire physical memory using DAMON, and provides simplified
 access monitoring results statistics, namely idle time percentiles and
 estimated memory bandwidth.
 
-.. _damon_stat_monitoring_accuracy_overhead
+.. _damon_stat_monitoring_accuracy_overhead:
 
 Monitoring Accuracy and Overhead
 ================================
-- 
2.39.5

